packages/manager/modules/bm-server-components/src/technical-details/technical-details.html

```diff
@@ -113,7 +113,7 @@
             ></span>
         </oui-tile-description>
         <oui-tile-description
-            data-ng-if="$ctrl.isDisksUpgradable() || $ctrl.upgradeTask.shouldScheduleUpgrade($ctrl.UPGRADE_TYPE.STORAGE)"
+            data-ng-if="$ctrl.showServiceActions && ($ctrl.isDisksUpgradable() || $ctrl.upgradeTask.shouldScheduleUpgrade($ctrl.UPGRADE_TYPE.STORAGE))"
```

Comment:
**nitpick:** I think it is better to add the check in `isRamUpgradable` & `isDisksUpgradable` to have better maintainability.

Reply:
I cannot add it in sRamUpgradable & isDisksUpgradable because I should not display the actions for second case as well ($ctrl.upgradeTask.shouldScheduleUpgrade($ctrl.UPGRADE_TYPE.STORAGE)).   There is a OR condition.